ipi/engine/motion/constrained_dynamics-old.py

```diff
@@ -0,0 +1,508 @@
+"""Contains the classes that deal with the different dynamics required in
```

Comment:
Should this file exist ?

Reply:
This is outdated and has now been removed.